README.md

```diff
@@ -565,14 +565,53 @@ assert.Len(t, arr, 3)
 ```go
 ❌
 assert.Less(t, a, 0)
+assert.Less(t, a, int(0))
```

Comment:
I appreciated that you follows guidelines, but I propose to do exception here.

just add to description 

```
**Reason**: More appropriate `testify` API with clearer failure message.

Typed zeros (like `int8(0)`) are also supported.

---

```

and remove such detailed examples from README and GoDoc, to make it more readable

Reply:
Sure, I will